### PR TITLE
update pd-client.

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

File: _vendor/vendor/github.com/pingcap/pd/pd-client/rpc_worker.go [hunk 1]
```diff
@@ -96,7 +97,8 @@ func (w *rpcWorker) stop(err error) {
 	close(w.quit)
 	w.wg.Wait()
 
-	for i := 0; i < len(w.requests); i++ {
+	n := len(w.requests)
+	for i := 0; i < n; i++ {
```
Comment (zimulala) [inline _vendor/vendor/github.com/pingcap/pd/pd-client/rpc_worker.go]: Line 104-111 can use the following form:
```case a,b,c,d:
```r.done <- err

**Comment by shenli** [comment]
LGTM

File: _vendor/vendor/github.com/pingcap/pd/pd-client/rpc_worker.go (hunk 1 above)
Comment (huachaohuang) [inline _vendor/vendor/github.com/pingcap/pd/pd-client/rpc_worker.go]: Yep, maybe I should change that later in PD, just update vendor here.

**Comment by zimulala** [comment]
LGTM

**Pull request merged**